fuse/utils/data/collate.py

```diff
@@ -34,13 +35,15 @@ def __init__(
         self,
         skip_keys: Sequence[str] = tuple(),
         raise_error_key_missing: bool = True,
+        missing_values: Sequence[str] = (None, "N/A")
```

Comment:
A documentation of this parameter is missing, tho I don't see a use of it so maybe it is supposed to be deleted?